week-02/week-02-group-learning.md

```diff
@@ -1 +1,102 @@
+# Week 02: Tooling for design- Organised by [ADD ORGANISERS' NAMES AND AFFILIATION AS NEEDED]!
+
+* Date: 
+* Time:
+* Duration: 90 min
+* Call Facilitators: 
+
+### Housekeeping 
+
+* Make a copy of this week’s template 
+* Download slides and resources needed for this week
+* Make sure you have access to this week’s resources (see week 02 info)
+* We will use some YouTube recordings to watch, either download them or make sure you have internet access to watch them online
+
+## This week’s objectives:
+
+- Identify tools to help you design your project in a clear way for newcomers to quickly understand what the project is about, who can be involved, and where the work is headed
+- Fill the open canvas for your project to think through an open project strategy to develop an MVP
+- Reflect on the roadmap for your project with milestones - start drafting them with descriptions and tasks
+- After this Week, develop documentation and resources for your project and share them with others
+
+
+## 🌍 Roll call
+
+### Introducing yourself
+
+*Name / Project/socials (where you'd like to connect with others / Icebreaker question: How have your first few weeks in OLS-9 been? Share links to a picture, a word, a gif to express how you are feeling about your role as a project lead and/or team member
+
+*
+*
+*
+*
+*
+*
+*
+*
+
+## 🗣️ Welcome!
+
+
+Quote of the week: “Design must be an innovative, highly creative, cross-disciplinary tool responsive to the needs of men. It must be more research-oriented, and we must stop defiling the earth itself with poorly-designed objects and structures.”
+― Victor Papanek, Design for the Real World: Human Ecology and Social Change
+
+[HOST] (10 min)[⏰ 10]
+
+* Code of conduct and community participation guidelines: [ORGANISERS TO ADD RELEVANT GUIDELINES HERE]
+
+If hosting this call under OLS, use: [https://we-are-ols.org/code-of-conduct](https://we-are-ols.org/code-of-conduct)
+
+  **Breakout room: Speaking and Writing**
+
+* Indicate by editing your name on Zoom and add “W” to indicate a preference for Writing, and “S” to indicate a preference for speaking. * This will help us assign you to the breakout room with the format and language of your choice
+  * Even if you are ok with both spoken and written reflections, please choose one option for this call to help us assign you easily to one group.
+
+## 🖥Tooling for design!
+
+[HOST] (15 min)[⏰ 25]
+
+* YouTube video link: https://www.youtube.com/watch?v=riBa5faSNIs&list=PL1CvC6Ez54KBD7zi7YSSS_L_WH87w5eo1&index=3 
+* Slides: https://doi.org/10.5281/zenodo.14268572 
+
+
+**Notes and self-reflection:**
+
+*
+*
+
+**What questions have emerged from this session that organisers can help you with**
+
+*
+*
```

Comment:
Are you sure this is the full cohort call text from the template?
I think a lot of sections are missing.